hr_course/models/hr_course.py

```diff
@@ -65,6 +76,62 @@ class HrCourse(models.Model):
         readonly=True,
     )
 
+    schedule_validity_end_date = fields.Date(
+        related="course_schedule_ids.validity_end_date"
```

Comment:
AFAIR, in this case, orm will try to assign validity_end_date from the first item

Reply:
i think outdated considering changes in the model.